test/acceptance/test/add_tests.go

```diff
@@ -279,18 +279,18 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 		})
 	})
 
-	It("Verify wego app add . --dry-run flag does not modify the cluster", func() {
+	It("SmokeTest - Verify 'wego app add' with --dry-run flag does not modify the cluster", func() {
 		var repoAbsolutePath string
 		var session *gexec.Session
 		private := true
-		branchName := "main"
+		branchName := "test-branch"
 		appManifestFilePath := "./data/nginx.yaml"
 		defaultSshKeyPath := os.Getenv("HOME") + "/.ssh/id_rsa"
 		appRepoName := "wego-test-app-" + RandString(8)
 		workloadName := "nginx"
 		workloadNamespace := "my-nginx"
 		url := "ssh://git@github.com/weaveworks-gitops-test/" + appRepoName + ".git"
```

Comment:
Should not hardcode the org part, instead should fetch from os.Getenv("GITHUB_ORG") environment variable.